README.md

```diff
@@ -70,11 +70,24 @@ Accessing demo Prometheus APIs on `127.0.0.1:8080` will now expect `tenant` quer
 
 ```bash
 ➜  ~ curl http://127.0.0.1:8080/api/v1/query\?query="up"
-Bad request. The "tenant" query parameter must be provided.
+The "tenant" query parameter must be provided.
 ➜  ~ curl http://127.0.0.1:8080/api/v1/query\?query="up"\&tenant\="something"
 {"status":"success","data":{"resultType":"vector","result":[]}}%
 ```
 
+You can also provide a static value for the tenancy label. For example, running `prom-label-proxy` with
```

Comment:
Tiny nit: we may want to clarify that any label can be set statically, not just a magic tenancy label.
```suggestion
You can also provide a static value for a label. For example, running `prom-label-proxy` with
```